Ports/qemu/package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='qemu'
+version='6.2.0'
```

Comment:
Does this simply pre-date the QEMU 7 release, or did you have some troubles with that?

Reply:
I _think_ I had some issues with dependencies, but with the recent changes we have been doing it might be worth it to uprev to 7.0.0. Will test later.

Reply:
(Actually, I might just have blindly copied from `BuildQemu.sh`, which likely was still at 6.2.0 at that time. In any case, I will check if 7.0.0 works for us.)